docs/status.md

```diff
@@ -47,7 +47,7 @@ one of the following tracking labels.
 | after_all                | yes           | yes          | no             | yes             | no          |
 | all_gather               | no            | revisit      | no             | no              | no          |
 | all_reduce               | no            | revisit      | revisit        | no              | no          |
-| all_to_all               | no            | yes*         | yes*           | no              | no          |
+| all_to_all               | yes           | yes          | yes            | no              | no          |
```

Comment:
Verification should be "revisit" because we aren't yet verifying `replica_groups` against `num_replicas` and `num_partitions`.